src/reaper_osara.cpp

```diff
@@ -153,6 +153,9 @@ string formatTime(double time, TimeFormat timeFormat, bool isLength,
 			timeFormat = TF_MEASURE;
 		}
 	}
+	if (!isLength && includeProjectStartOffset && timeFormat != TF_MEASURE && timeFormat != TF_SAMPLE) {
```

Comment:
I'm not 100% sure about TF_SAMPLE here, but I don't think applying the project start time to samples is a good idea.